Anytype/Sources/PresentationLayer/Modules/Membership/Views/NameValidation/MembershipNameValidationViewModel.swift

```diff
@@ -61,7 +61,7 @@ final class MembershipNameValidationViewModel: ObservableObject {
             do {
                 try await membershipService.validateName(name: name, tierType: tier.type)
                 state = .validated
-            } catch let error {
+            } catch let error as MembershipServiceProtocol.ValidateNameError {
```

Comment:
I think is better for ignore cancellation error, because middleware can return a lot of different error and user should be visible any error. Alwe we will add a timeout error. It should be visible for user